docs/install/tutorials/uwsgi.rst.draft

```diff
@@ -0,0 +1,15 @@
+.. _how_to_uwsgi:
```

Comment:
I think we could replace this section by advicing the installation of kolibri-server. If the instructions to install kolibri are followed, installing kolibri-server is as simple as 
`sudo apt-get install kolibri-server`
The package is now in the kolibri-proposal ppa but will be in the kolibri ppa as soon as 0.12 is released.

Obviously these instructions are valid, but quite more complex.

Reply:
This was in a `.draft` file, so not actually in the docs.

Reply:
If it's a draft that will be superseded anyway, might make more sense to just remove it?

Reply:
It contained some text that I feel could be reused.